source/retryable-reads/tests/README.rst

```diff
@@ -148,14 +148,17 @@ data.
 .. _GridFSBucket spec: https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst#configurable-gridfsbucket-class
     
 Speeding Up Tests
------------------
+===================
```

Comment:
This should line up with the heading text, but I also don't see any reason this needs to change. The README already had a section for this, so we can leave it at its current nesting level, since this applies within the "Network Error Tests" section.

Reply:
@DmitryLukyanov ping